actors/builtin/market/market_actor.go

```diff
@@ -346,10 +346,13 @@ func (a Actor) ActivateDeals(rt Runtime, params *ActivateDealsParams) *adt.Empty
 				rt.Abortf(exitcode.ErrIllegalArgument, "deal %d already included in another sector", dealID)
 			}
 
-			proposal, err := proposals.Get(dealID)
+			proposal, found, err := proposals.Get(dealID)
 			if err != nil {
 				rt.Abortf(exitcode.ErrIllegalState, "get deal %v", err)
 			}
```

Comment:
Nit: use `RequireNoErr` here (and include the deal id).

Reply:
This is done.